composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5",
```

Comment:
Are u saying I should not bumping requirement in this PR until 0.5 is done? Or just pointing out 0.5 work is still ahead?